code/build/template/data/[+python python]/orx/guid.pyi

```diff
@@ -0,0 +1,4 @@
+class Guid:

```

Comment:
This is a custom wrapper type since Python does not have a concept of unsigned types, and pocketpy uses `int64_t` internally for Python `int` values.